parsel/selector.py

```diff
@@ -10,6 +10,9 @@
 from .utils import flatten, iflatten, extract_regex, shorten
 from .csstranslator import HTMLTranslator, GenericTranslator
 
+import jmespath
+import json
```

Comment:
:lipstick: 
- `import json` should be right above `import sys`, `json` being a built-in module
- `import jmespath` should be above `import six`

Reply:
thank you! it's my fault! i will fix in next time!